x/superfluid/keeper/migrate_test.go

```diff
@@ -1230,179 +1231,195 @@ type positionInfo struct {
+		for _, positionInfo := range positionInfos {
+			for i, posInfo := range positionInfo {
+				balancerPool, err = s.App.GAMMKeeper.GetCFMMPool(s.Ctx, balancerPoolId)
+				s.Require().NoError(err)
 
-			// Note owner and balancer pool balances before migration.
-			preClaimOwnerBalance := s.App.BankKeeper.GetAllBalances(s.Ctx, s.TestAccs[i+1])
-			preClaimBalancerPoolBalance := balancerPool.GetTotalPoolLiquidity(s.Ctx)
+				// Note owner and balancer pool balances before migration.
+				preClaimOwnerBalance := s.App.BankKeeper.GetAllBalances(s.Ctx, s.TestAccs[i+1])
+				preClaimBalancerPoolBalance := balancerPool.GetTotalPoolLiquidity(s.Ctx)
 
-			// Run the migration.
-			_, amount0, amount1, _, _, _, _, err := s.App.SuperfluidKeeper.RouteLockedBalancerToConcentratedMigration(s.Ctx, s.TestAccs[i+1], posInfo.lockId, posInfo.coin, sdk.Coins{})
-			s.Require().NoError(err)
+				// Run the migration.
+				_, amount0, amount1, _, _, _, _, err := s.App.SuperfluidKeeper.RouteLockedBalancerToConcentratedMigration(s.Ctx, s.TestAccs[i+1], posInfo.lockId, posInfo.coin, sdk.Coins{})
+				s.Require().NoError(err)
 
-			// Note how much of amount0 and amount1 was actually created in the CL pool from the migration.
-			clJoinPoolAmt := sdk.NewCoins(sdk.NewCoin(clPool.GetToken0(), amount0), sdk.NewCoin(clPool.GetToken1(), amount1))
+				// Note how much of amount0 and amount1 was actually created in the CL pool from the migration.
+				clJoinPoolAmt := sdk.NewCoins(sdk.NewCoin(clPool.GetToken0(), amount0), sdk.NewCoin(clPool.GetToken1(), amount1))
 
-			// Note owner and balancer pool balances after migration.
-			balancerPool, err = s.App.GAMMKeeper.GetCFMMPool(s.Ctx, balancerPoolId)
-			s.Require().NoError(err)
-			postClaimOwnerBalance := s.App.BankKeeper.GetAllBalances(s.Ctx, s.TestAccs[i+1])
-			postClaimBalancerPoolBalance := balancerPool.GetTotalPoolLiquidity(s.Ctx)
-
-			// Check that the diff between the initial balancer position and the new CL position is equal to the amount that was left behind in the balancer pool and the amount that was sent back to the owner.
-			for i := range posInfo.joinPoolCoins {
-				balancerToClBalanceDelta := posInfo.joinPoolCoins[i].Amount.Sub(clJoinPoolAmt.AmountOf(posInfo.joinPoolCoins[i].Denom))
-				userBalanceDelta := postClaimOwnerBalance.AmountOf(posInfo.joinPoolCoins[i].Denom).Sub(preClaimOwnerBalance.AmountOf(posInfo.joinPoolCoins[i].Denom))
-				balancerPoolDelta := posInfo.joinPoolCoins[i].Amount.Sub(preClaimBalancerPoolBalance.AmountOf(posInfo.joinPoolCoins[i].Denom).Sub(postClaimBalancerPoolBalance.AmountOf(posInfo.joinPoolCoins[i].Denom)))
-				if i == 0 {
-					totalBalancerPoolFundsLeftBehindAmount0 = totalBalancerPoolFundsLeftBehindAmount0.Add(balancerPoolDelta)
-				} else {
-					totalBalancerPoolFundsLeftBehindAmount1 = totalBalancerPoolFundsLeftBehindAmount1.Add(balancerPoolDelta)
+				// Note owner and balancer pool balances after migration.
```

Comment:
Note to other reviewers: No notable changes here, its just github failing to parse diffs correctly